samples/javascript_nodejs/84.bot-authentication-certificate/.env

```diff
@@ -1,4 +1,3 @@
-MicrosoftAppTenantId=
```

Comment:
I noticed we dont have the MicrosoftAppTenantId and MicrosoftAppType, whe should include them here in the .env and the index.js, because they could be used with this flow path.

Reply:
https://github.com/microsoft/botbuilder-js/blob/066b405a8630e4f6c7d165ef939982181f6bbc6a/libraries/botbuilder-core/src/configurationBotFrameworkAuthentication.ts#L28

MicrosoftAppType doesn't exist in ConfigurationBotFrameworkAuthenticationOptions

![image](https://github.com/southworks/BotBuilder-Samples/assets/122501764/be283776-9f52-4ac6-86ec-a65d1c169eca)

Reply:
Also does not exist in CertificateServiceClientCredentialsFactory(SN+I)
https://github.com/microsoft/botbuilder-js/blob/066b405a8630e4f6c7d165ef939982181f6bbc6a/libraries/botframework-connector/src/auth/certificateServiceClientCredentialsFactory.ts#L41

Reply:
added Tenant Id

Reply:
The app type should be added as well, i know its not in the ConfigurationBotFrameworkAuthenticationOptions, but it is in the https://github.com/microsoft/botbuilder-js/blob/066b405a8630e4f6c7d165ef939982181f6bbc6a/libraries/botbuilder-core/src/configurationServiceClientCredentialFactory.ts which is the one that instantiates the CertificateServiceClientCredentialsFactory